CMakeLists.txt

```diff
@@ -296,14 +296,25 @@ elseif(UNIX)
 
   if(APPLE)
     set(CMAKE_MACOSX_RPATH 1)
-    target_sources(PortAudio PRIVATE
-      src/hostapi/coreaudio/pa_mac_core.c
-      src/hostapi/coreaudio/pa_mac_core_blocking.c
-      src/hostapi/coreaudio/pa_mac_core_blocking.h
-      src/hostapi/coreaudio/pa_mac_core_internal.h
-      src/hostapi/coreaudio/pa_mac_core_utilities.c
-      src/hostapi/coreaudio/pa_mac_core_utilities.h
-    )
+    if(IOS)
```

Comment:
At some point, we will need a GitHub Actions for Continuous Integration testing for iOS.

Reply:
I have added an experimental iOS matrix configuration (which is pretty similar to macOS)